src/Magnum/Math/Test/DualQuaternionTest.cpp

```diff
@@ -469,6 +471,17 @@ void DualQuaternionTest::translation() {
     CORRADE_COMPARE(q.translation(), vec);
 }
 
+void DualQuaternionTest::rotationTranslation() {
+    Vector3 axis(1.0f/Constants<Float>::sqrt3());
+    Quaternion r = Quaternion::rotation(Deg(120.0f), axis);
```

Comment:
```suggestion
    Quaternion r = Quaternion::rotation(120.0_degf, axis);
```

:trollface:

Reply:
Copied this from another test, sorry :D

Reply:
Yeah I need to go over all those and clean that up. It was originally for compat with GCC 4.6 which didn't have UDL support yet, but those days are fortunately long gone.

Reply:
I replaced all `Deg(<n>)` with `<n>_degf` in the DualQuaternionTest in a separate commit